ipwb/__main__.py

```diff
@@ -131,11 +131,14 @@ def checkArgs(argsIn):
 
     argCount = len(argsIn)
     cmdList = ['index', 'replay']
+    baseParserFlagList = ['-d', '--daemon', '-v', '--version']
 
     # Various invocation error, used to show appropriate help
     cmdError_index = argCount == 2 and argsIn[1] == 'index'
     cmdError_noCommand = argCount == 1
-    cmdError_invalidCommand = argCount > 1 and argsIn[1] not in cmdList
+    cmdError_invalidCommand = argCount > 1 \
+        and argsIn[1] not in cmdList \
+        and argsIn[1] not in baseParserFlagList
```

Comment:
These two lines can perhaps be simplified as:

```python
and argsIn[1] not in cmdList + baseParserFlagList
```

Reply:
It may but I recall running into some issues. Your suggestion has been applied, though a line break was still needed for pycodestyle compliance.

Reply:
This was a non-blocking stylistic change. Please do make sure that it works as expected after making this change.

Reply:
@ibnesayeed I did so before applying the changes. Joining lists/arrays via `+` makes we wary but I suppose it's a standard Python construct.